pkgs/development/python-modules/disposable-email-domains/default.nix

```diff
@@ -0,0 +1,37 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  setuptools,
+  pytestCheckHook,
+}:
+
+buildPythonPackage rec {
+  pname = "disposable-email-domains";
+  version = "0.0.126";
+  pyproject = true;
+
+  # No tags on GitHub
+  src = fetchPypi {
+    pname = "disposable_email_domains";
+    inherit version;
+    hash = "sha256-fPxAOOT7yIT7DoupnUlC6iQQWBMw5iaNoH8hjjWBKMI=";
+  };
+
+  build-system = [
+    setuptools
+  ];
+
+  pythonImportsCheck = [ "disposable_email_domains" ];
+
+  nativeCheckInputs = [ pytestCheckHook ];
+
+  meta = {
+    description = "Set of disposable email domains";
+    homepage = "https://github.com/disposable-email-domains/disposable-email-domains";
+    # The GitHub mentions CC-0 (https://github.com/disposable-email-domains/disposable-email-domains/blob/main/LICENSE.txt),
+    # but Pypi advertises MIT.
+    license = lib.licenses.mit;
```

Comment:
I think we should go with the license that is next to the source code. The section in the readme suggests CC-0 wasn't an oversight either.